ReactWindows/ReactNative.Shared/DevSupport/DevServerHelper.cs

```diff
@@ -7,8 +7,10 @@
 using System.Reactive.Disposables;
 using System.Threading;
 using System.Threading.Tasks;
+#if WINDOWS_UWP
 using Windows.System.Threading;
 using Windows.UI.Core;
```

Comment:
Are both of these namespaces needed for UWP now that you changed from ThreadPool.Run -> Task.Run?